### PR TITLE
feat(firestore): adds COUNT query sample

### PR DESCRIPTION
**Pull request by telpirion**

### CODE REVIEW

File: firestore/cloud-client/aggregate_query_count_test.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+os.environ["GOOGLE_CLOUD_PROJECT"] = os.environ["FIRESTORE_PROJECT"]
+PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
+
+
+def test_create_count_query(capsys):
+    create_count_query(project_id=PROJECT_ID)
+    out, _ = capsys.readouterr()
+
+    assert "Number of" in out
```
Comment (Mariatta) [inline firestore/cloud-client/aggregate_query_count_test.py:28]: Can you also assert the alias?

Comment (Mariatta) [reply firestore/cloud-client/aggregate_query_count_test.py:28]:
```suggestion
    assert "Alias of results from query: all" in out
    assert "Number of results from query: <somenumber>" in out
```

**Comment by Mariatta** [comment]
For count query in datastore, we added samples for using it with transaction and snapshot. Would those worth adding as samples in Firestore to? https://github.com/googleapis/python-datastore/pull/383/files

File: firestore/cloud-client/aggregate_query_count_test.py (hunk 1 above)
Comment (telpirion) [reply firestore/cloud-client/aggregate_query_count_test.py:28]: Done.

**Review by Mariatta — APPROVED** [review]

**Pull request merged**